build.gradle

```diff
@@ -9,7 +9,17 @@ buildscript {
 }
 
 apply plugin: 'com.mparticle.kit'
+apply plugin: 'com.android.library'
```

Comment:
I don't think applying this plugin is necessary - com.mparticle.kit will do it for you